js/controllers/window.js

```diff
@@ -27,6 +28,12 @@ function WindowController(editor, settings, analytics, tabs) {
   }
 }
 
+/** Initializes UI elements and animations. */
```

Comment:
Actually, I feel like you might as well inline this call. It's not useful to break the linearity of your code for a few lines, in my opinion.

Reply:
I would inline it for now, and when it does get too long we can consider breaking it out. Right now, I think having a single comment would be sufficient (e.g. something like `// Initialize Material UI components.`).

Reply:
SGTM, done!